src/content/app/tools/blast/views/blast-form/BlastForm.tsx

```diff
@@ -27,8 +27,58 @@ import ToolsTopBar from 'src/content/app/tools/shared/components/tools-top-bar/T
 import BlastInputSequencesHeader from 'src/content/app/tools/blast/components/blast-input-sequences/BlastInputSequencesHeader';
 import BlastInputSequences from 'src/content/app/tools/blast/components/blast-input-sequences/BlastInputSequences';
 
+import SpeciesSelectorHeader from 'src/content/app/tools/blast/components/species-selector/SpeciesSelectorHeader';
+import SpeciesSelector from 'src/content/app/tools/blast/components/species-selector/SpeciesSelector';
+
 import styles from './BlastForm.scss';
 
+//TODO: hardcoding species list for now, this need replacing with API when we implement search or add more species and find out which field we need to submit a job
+const speciesList = [
+  {
+    assembly_name: 'GRCh38.p13',
+    common_name: 'Human',
+    genome_id: 'homo_sapiens_GCA_000001405_28',
+    scientific_name: 'Homo sapiens'
+  },
+  {
+    assembly_name: 'IWGSC',
+    common_name: null,
+    genome_id: 'triticum_aestivum_GCA_900519105_1',
+    scientific_name: 'Triticum aestivum'
+  },
+  {
+    assembly_name: 'GRCh37.p13',
+    common_name: 'Human',
+    genome_id: 'homo_sapiens_GCA_000001405_14',
+    scientific_name: 'Homo sapiens'
+  },
+  {
+    assembly_name: 'WBcel235',
+    common_name: null,
+    genome_id: 'caenorhabditis_elegans_GCA_000002985_3',
+    scientific_name: 'Caenorhabditis elegans'
+  },
+  {
+    assembly_name: 'R64-1-1',
+    common_name: null,
+    genome_id: 'saccharomyces_cerevisiae_GCA_000146045_2',
+    scientific_name: 'Saccharomyces cerevisiae'
+  },
+  {
+    assembly_name: 'ASM584v2',
+    common_name: null,
+    genome_id:
+      'escherichia_coli_str_k_12_substr_mg1655_gca_000005845_GCA_000005845_2',
+    scientific_name: 'Escherichia coli str. K-12 substr. MG1655'
+  },
+  {
+    assembly_name: 'ASM276v2',
+    common_name: null,
+    genome_id: 'plasmodium_falciparum_GCA_000002765_2',
+    scientific_name: 'Plasmodium falciparum 3D7'
+  }
+];
```

Comment:
Let's move this out of the component into its own file, and import it directly into the species selector component. If you like the code for sorting the species, move it over into that file as well, sort the species in that file, and export a sorted list of species;

Reply:
Done

Reply:
> One question about this, are we sure we dont want to pass the speciesList as props

Looking at the [design](https://xd.adobe.com/view/879a4244-8420-48c1-b3c9-c998e77651fd-d2c9/screen/8dffb6f7-a101-4758-9870-18143352b5e1?fullscreen) of the future species selector for BLAST, the  most likely evolution of this component will be something like the following:

- There will be a `BlastSpeciesSelector` wrapper component containing the functions that are specific to the BLAST form (e.g. connecting with the blastFormSlice). Let's call it component 1.
- It will contain several children, likely to be shared with species selectors for other tools:
  - A component with a search field (let's call it component 2).
  - A component with a list of search results, with checkboxes to select species (let's call it component 3)
  - A component with a list of added species, with delete icons (let's call it component 4)
- These children components will be receiving the lists of selected species from the parent

Your temporary component is now a combination of component 1 and component 3. Component 1 will connect to redux directly and will receive the list of species from there rather than from the BlastForm component. So I wouldn't make the BlastForm component responsible for passing the list of species to your temporary species selector even now.

Reply:
Done